### PR TITLE
Adding common repo files

### LINKED ISSUE #17: License file
A proper license file needs to be added and linked in the main README.

The license to use is the Apache Version 2.0 license.

### PR DESCRIPTION
**Pull request by mbrandenburger**
Adding common files to the repo as suggested for all Hyperledger repos.

Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

This fixes #17

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hyperledger Fabric Private Chaincode
```
Comment (matejpavlovic) [inline NOTICE:1]: MirBFT

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
```
Comment (matejpavlovic) [inline CODE_OF_CONDUCT.md:4]: CC or Apache? Just this file or others too?

**Review by matejpavlovic — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (mbrandenburger) [reply CODE_OF_CONDUCT.md:4]: oh even here this is not needed. removed

**Review by matejpavlovic — APPROVED** [review]
Looks good to me!

**Pull request merged**